README.md

```diff
@@ -4,27 +4,43 @@ Automating MakePlayingCards's online ordering system.
+   1. It might make the most sense to unzip the Elasticsearch directory into your `$working_directory`
+   2. Optionally, add the executables to your PATH for easy terminal access (add `location-of-package/elasticsearch-<VERSION_NUM>/bin` to your PATH)
+7. Run Elasticsearch
+   1. Use the executable indicated in your OS's Installation Guide, optionally running it as a daemon.
+   2. Test that Elasticsearch is running by querying the local instance. For example: `wget -qO- http://localhost:9200/`
+8. Set up your Google Drive credentials using a Google Drive service account. For each step, only follow the instructions for the given section, then return and complete the next step. The linked tutorial is slightly out of date, but it is the most complete tutorial around.
+   1. [Step 1](https://help.talend.com/r/E3i03eb7IpvsigwC58fxQg/uEUUsDd_MSx64yoJgSa1xg) - NOTE: The project name, product name, and Application type don't matter for just running the tool locally.
+   2. [Step 2](https://help.talend.com/r/E3i03eb7IpvsigwC58fxQg/ol2OwTHmFbDiMjQl3ES5QA) - NOTE: The JSON file you download at the end of this step is named something slightly different from what the application expects. Make sure to rename it.
+   3. Rename the saved JSON file and move it to: `$working_directory/MPCAutofill`
+9. Update the `$working_directory/MPCAutofill/drives.csv` spreadsheet with links to public Google Drives with MPC ready images
+10. Install the main dependencies using a python virtual environment (this entire step is often automatically handled by your IDE)
+    1. To initialize the environment, create the virtual environment library using `python3 -m venv $working_directory/venv`
+    2. Activate the environment using your OS specific command in the table found in this section of the docs: [Scroll to the table](https://docs.python.org/3.8/library/venv.html#creating-virtual-environments)
+    3. In the `$working_directory` with the virtual environment active, run the command `pip3 install -r requirements.txt` to install all requirements listed in `$working_directory/requirements.txt`
+11. In the `$working_directory`, run the command `python manage.py migrate` to ensure the database tables reflect any Django model changes.
+12. In the `$working_directory`, run the command `python manage.py update_dfcs` to synchronise the double-faced cards table with Scryfall.
+13. In the `$working_directory`, run the command `python manage.py import_sources` to sync sources in `$working_directory/MPCAutofill/drives.csv` to the database (optionally specifying a particular drive to sync with `-d <drivename>`)
+14. (Elasticsearch must be running for this script to work)  
+   In the `$working_directory`, run the command `python manage.py update_database` to populate the database with cards from the sources loaded with `import_sources`
```

Comment:
For steps 10.3 - 14 it should read: In `$working_directory/MPCAutofill`
(it didn't work on windows unless you were in the same directory as the manage.py file)

Reply:
Changes implemented